### ISSUE TITLE
Can't start newly created rekit project

### ISSUE THREAD

**Issue by shockey** [supnate/rekit#52]
### Context

macOS High Sierra
node@6.11.3
npm@3.10.10

Created a rekit application like this:
```
➜ npm i -g rekit
➜ rekit create 12n-frontend
➜ cd 12n-frontend
➜ git init .
➜ git add . && git commit -m "init"
➜ npm i
➜ npm start
```

Rekit package versions:

```
➜  npm ls | grep rekit
├─┬ rekit-core@2.0.1
├─┬ rekit-portal@2.0.6
```

### Error

Upon running `npm start`, I get:

```
> 12n-frontend@0.0.0 start
> node ./tools/server.js

module.js:471
    throw err;
    ^

Error: Cannot find module '/Users/i305656/workspace/a2/package.json'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.loadPlugins (node_modules/rekit-core/core/plugin.js:49:22)
    at Object.<anonymous> (node_modules/rekit-portal/middleware/index.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
```

Not quite sure what to make of this. Any pointers?

**Update**: I worked around this problem by downgrading my `rekit-portal` to 2.0.5.

**Comment by shockey** [comment]
Upon further inspection, I think this commit in `rekit-core` is the culprit: https://github.com/supnate/rekit-portal/commit/e698b8f057d37ac44fc20c6151798be9c318fbcb#diff-f15dd4b6fc7489eaab987b413919b2eaR18

**Comment by supnate** [comment]
Fixed in rekit-portal 2.0.7, sorry for the stupid bug.

**Comment by shockey** [comment]
@supnate, no worries! Thanks for these wonderful tools 😄
